vue2/src/views/Home.vue

```diff
@@ -17,14 +29,12 @@
 
 <script>
 import { years, monthDays } from '../helpers/definition';
-console.log(monthDays);
 
 export default {
   data () {
     return {
       years:years,
```

Comment:
前回の指摘漏れになりますが、`:`のあとにはスペースを空けるようにしましょう。
他の部分も同様です。
```suggestion
      years: years,
```